lib/vmdb/console_methods.rb

```diff
@@ -28,6 +28,7 @@ def simulate_queue_worker(break_on_complete = false)
         else
           break_on_complete ? break : sleep(1.second)
```

Comment:
I think this will bite someone...as in they want to wait for the next 5 messages even if they haven't arrived yet, but let's see how it goes.